src/main/java/org/isf/menu/gui/MainMenu.java

```diff
@@ -348,61 +358,71 @@ private void launchApp(String itemMenuCode) {
 		private static final long serialVersionUID = 4338749100837551874L;
 
 		private JButton[] button;
-		private MainMenu parentFrame;
+		private MainMenu parentFrame = null;
 
 		public MainPanel(MainMenu parentFrame) {
 			this.parentFrame = parentFrame;
 			int numItems = 0;
 
-			for (UserMenuItem u : myMenu) {
-				if (u.getMySubmenu().equals("main")) {
+			for (UserMenuItem u : myMenu)
+				if (u.getMySubmenu().equals("main"))
 					numItems++;
-				}
-			}
+
+			// System.out.println(numItems);
+
```

Comment:
new standard on develop is to have explicit code blocks {} for loops and if statements and to remove commented System.out.println

Reply:
restored current informatici version